config/spryk/spryks/Spryker/Zed/Test/DataBuilder/AddDataBuilderProperty.yml

```diff
@@ -0,0 +1,31 @@
+name: AddDataBuilderProperty
```

Comment:
Is there a use case for adding databuider property without the same property into the original transfer?
Could it be one operation: "I want to add transfer property and want that the same databuilder property will be also generated" ?

Reply:
It is exactly like this. The `AddDataBuilderProperty` is a `postSpryk` of `AddSharedTransferProperty`. You can still use it standalone as any other Spryk.